modules/nf-core/dragmap/align/environment.yml

```diff
@@ -3,8 +3,8 @@
 channels:
   - conda-forge
   - bioconda
-
 dependencies:
-  - dragmap=1.3.0
-  - pigz=2.8
-  - samtools=1.18
+  # WARN: Do not update this tool to 1.3.0 until https://github.com/Illumina/DRAGMAP/issues/47 is resolved
+  - dragmap=1.2.1
+  - pigz=2.3.4
+  - samtools=1.19.2
```

Comment:
We usually add the channel prefix? Does linting not complain about that?

Reply:
nope